magento/pwa-studio
# [Doc] Release notes 11.0.0

## Description

Update the changelog with release notes for the 11.0.0 release.

Also updates the compatibility table.

## Related Issue







Closes PWA-1875

## Acceptance

@davemacaulay 
@tkacheva 

### Verification Stakeholders

Any core developer

@davemacaulay 
@tkacheva 

### Specification




## Test Plan





1. Open CHANGELOG.md
2. Verify content is accurate
3. Open magento-compatibility.js
4. Verify content is accurate

#### Test scenario(s) for direct fix/feature
Example
-   Verify user is able to apply filters on category page
-   Verify user is able to apply filters on search page

#### Test scenario(s) for any existing impacted features/areas
Example
-   Verify user is able to sort data after applying filters on category page
-   Verify user is able to sort data after applying filters on search page

#### Test scenario(s) for any Magento Backend Supported Configurations
Example
- Update default Sort value in backend and repeat above scenarios

#### Is Browser/Device testing needed?
Example
- Yes, browser testing is needed as X UI component may be impacted on <browser>
- Yes, device testing is needed as X UI component may be impacted on <mobile|desktop|etc>

#### Any ad-hoc/edge case scenarios that need to be considered?
Example
- Apply all filters to get 0 results and then remove filters to see respective products 

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
